pom.xml

```diff
@@ -35,15 +40,38 @@
     <dependency>
       <groupId>com.esri.geoevent.sdk</groupId>
       <artifactId>geoevent-sdk</artifactId>
-      <version>${project.version}</version>
+      <version>[{geoevent.compat.version},)</version>
```

Comment:
As noted above, this preserves backward compatibility and has no down side that I know of. I look forward to comment from @eironside as well.

If you reject these two changes, then I will need to set the package version to 10.7.1, as it is critical that it works at least back to that version. In that case, unless there is a reason not to do this, I'd still prefer to use the [v,) notation to specify the minimum required SDK version as 10.7.1, and indicate that later versions are also acceptable (even though it will probably work with later versions of GeoEvent Server even without that).

Reply:
I am keeping this part unchanged in my branch for now until I understand better from @eironside about the standard.

Reply:
Actually I changed the project version to 10.4.1 and removed geoevent.compat.version. The explicit [v,) around the version number is recommended in the maven docs, even though the implicit behavior for v does seem to behave the same in GeoEvent Server. I tested this in GeoEvent Server 10.7.1 and confirmed it works. I have not yet tested with 10.8.1.